public/app/AppWrapper.tsx

```diff
@@ -85,35 +87,47 @@ export class AppWrapper extends React.Component<AppWrapperProps, AppWrapperState
 
     const newNavigationEnabled = Boolean(config.featureToggles.newNavigation);
 
+    const commandPaletteActionSelected = (action: Action) => {
```

Comment:
@meanmina and @davkal both chatted with me about feature tracking for this. I've added two individual events, one for opening the command palette and one when an action is selected. I think ideally what we might want here is something more like task tracking - generate a unique ID when the command palette is opened, and tie that ID to either an action selected event or an action cancelled event depending on the outcome. 

This specific case is simple but for more complicated tasks, we've used that to measure not just how much a feature is used, but how long a user spent trying to perform a specific action so we could prioritize what might need more UX help. Not sure if that's on our radar yet but might be something useful eventually. Feel free to hit me up for more info about this kinda thing.